src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5584,6 +5584,25 @@ CompareTasksByTaskId(const void *leftElement, const void *rightElement)
 	return taskIdDiff;
 }
 
+/* Helper function to compare two tasks by their execution duration. */
```

Comment:
```c
/*
 * CompareTasksByExecutionDuration is a helper function ..
 */
```

We start with name of the function and do not make the comment a one-liner when commenting a function